src/ansys/dpf/gate/property_field_grpcapi.py

```diff
@@ -24,6 +24,16 @@ def csproperty_field_new_on_client(client, numEntities, data_size):
         request.datatype = "int"
         return field_grpcapi._get_stub(client).Create(request)
 
+    @staticmethod
+    def csproperty_field_new_location_on_client(client, numEntities, data_size, location):
```

Comment:
why did we need a new API here, couldn't we do field_def_new and then set_location?